tests/workflows/components/workflows.libsonnet

```diff
@@ -138,7 +138,7 @@
         },
         spec: {
           entrypoint: "e2e",
-          ttlSecondsAfterFinished: 3600,
+          ttlSecondsAfterFinished: 8*60*60,
```

Comment:
Please calculate the number directly, leave `// xx days/hrs `

Reply:
BTW, is it better to change Argo configuration in CI cluster directly? Then we don't have change it per job

Reply:
Emmm don't you think `8*60*60` might make more sense to users, otherwise, they need to calculate how long is 28800 seconds, I think it make sense to add comment like `//8hours`

Reply:
This is minor issue. I am fine.